src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -238,7 +240,7 @@ public async Task<MutableConflictResolution> ResolveConflictsAsync()
                     {
                         var definitionLocations = _renameLocationSet.Symbol.Locations;
                         var definitionDocuments = definitionLocations
-                            .Select(l => conflictResolution.OldSolution.GetRequiredDocument(l.SourceTree))
+                            .Select(l => conflictResolution.OldSolution.GetRequiredDocument(l.SourceTree!))
```

Comment:
i Have no idea why, but changing line 121 caused an NRT warning here...

Reply:
@CyrusNajmabadi So I had (not yet submitted in a PR) a change that was also touching SolutionServices and I think I saw this behavior too.

Reply:
Did you figure out what it was?